tests/components/songpal/test_config_flow.py

```diff
@@ -155,6 +119,22 @@ async def test_flow_import(hass):
     mocked_device.get_interface_information.assert_not_called()
 
 
+async def test_flow_import_without_name(hass):
+    """Test import flow without optional name."""
+    mocked_device = _create_mocked_device()
+
+    with _patch_config_flow_device(mocked_device), _patch_setup():
+        result = await hass.config_entries.flow.async_init(
+            DOMAIN, context={"source": SOURCE_IMPORT}, data={CONF_ENDPOINT: ENDPOINT}
+        )
+        assert result["type"] == RESULT_TYPE_CREATE_ENTRY
+        assert result["title"] == MODEL
+        assert result["data"] == {CONF_NAME: MODEL, CONF_ENDPOINT: ENDPOINT}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)